Leafly-com/lm-review-service
# Lmd 472 create review

## Description

Adds `/v1/' handler to create reviews + happy path unit tests

## Manual Test

- docker-compose up -d
- npm run start
- npm run build
- `POST http://localhost:8080/v1/`
```
{
	"email": "hello@email.com",
	"sku": "111",
	"type": "comment",
	"text": "hi",
	"rating": 3,
	"published": true,
	"language": "US"
}
```

Confirm Mongo contains the review:
![image](https://user-images.githubusercontent.com/54331819/73025051-80c61080-3df4-11ea-9a16-f3f1ea2382be.png)

## Unit Tests

![image](https://user-images.githubusercontent.com/54331819/73025100-950a0d80-3df4-11ea-9216-9168ecea8d4c.png)
